vertica_python/vertica/messages/frontend_messages/startup.py

```diff
@@ -95,6 +95,7 @@ def __init__(self, user, database, session_label, os_user_name, autocommit,
             b'binary_data_protocol': '1' if binary_transfer else '0', # Defaults to text format '0'
             b'protocol_features': '{"request_complex_types":' + request_complex_types + '}',
             b'protocol_compat': 'VER',
+            b'workload': workload,
```

Comment:
Actually since we are just querying the dc client server messages table it won't require a protocol version increase. I'll remove this

Reply:
This also requires "SHOW WORKLOAD" which I've implemented on the server in version 23.3. This test can't pass until we have a Vertica image available for 23.3 which is not yet released so it will be a while.